cts/CIB.py

```diff
@@ -467,7 +467,8 @@ def __init__(self, CM):
         self.register("pacemaker12", CIB12, CM, self)
         self.register("pacemaker20", CIB20, CM, self)
 #        self.register("hae", HASI, CM, self)
-        self.target = self.CM.Env["nodes"][0]
+        if self.CM.Env["ListTests"] == 0:
+            self.target = self.CM.Env["nodes"][0]
```

Comment:
Whoops, I missed this for 2.0. I assume without this, the array is empty and raises an exception ...

Looking at the rest of the code, it seems this class often depends on self.target being set. Rather than skip just this line, I'm thinking CTSlab.py should skip instantiating this class at all for ListTests. I have a 2.0 PR ready to merge, so I'll do up a quick commit to add that, and then you can backport that change instead of the above. I'll comment again here when it's ready.

Reply:
Argh, ListTests does need to instantiate crm_corosync, to test whether a test is applicable. In any case, I'll create a new commit for 2.0.

Reply:
I'm just going to duplicate your fix here for 2.0 for now, so we can merge this. Haven't had time to dig further.